lightning_examples/distributed-training/ddp_example_5.py

```diff
@@ -0,0 +1,117 @@
+import torch
```

Comment:
each example has to be be in own folder

Reply:
this will not be shown, it is just for convenience when we need to rerun some of the ddp example when modifications are needed.

(the ddp examples cannot run inside the notebook directly)